icicle/appUtils/ntt/ntt.cu

```diff
@@ -520,7 +520,7 @@ namespace ntt {
    *  - `S` is the [scalar field](@ref scalar_t) of the curve;
    */
   extern "C" cudaError_t
-  CONCAT_EXPAND(CURVE, InitializeDomain)(curve_config::scalar_t primitive_root, device_context::DeviceContext& ctx)
+    CONCAT_EXPAND(CURVE, InitializeDomain)(curve_config::scalar_t primitive_root, device_context::DeviceContext& ctx)
```

Comment:
Formatting

Reply:
Which version of clang-format are you/CI machine using?

Reply:
locally I'm running 14.0.0.
the github hosted runners are running the software provided here https://github.com/actions/runner-images/blob/ubuntu22/20240107.1/images/ubuntu/Ubuntu2204-Readme.md#installed-software